httpbakery/discharge.go

```diff
@@ -123,3 +125,10 @@ func randomBytes(n int) ([]byte, error) {
 	}
 	return b, nil
 }
+
+// MkHandler converts an httprouter.Handle into an http.Handler.
```

Comment:
This doesn't belong in httpbakery - its use of httprouter types is entirely an internal implementation thing. If you use httprouter in the examples, you won't need this there anyway, I think.